src/main/java/se/sundsvall/contactsettings/api/model/Delegate.java

```diff
@@ -0,0 +1,66 @@
+package se.sundsvall.contactsettings.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.Objects;
+
+@Schema(description = "Delegate model")
+public class Delegate {
+
+	@Schema(description = "Unique id for the contact setting", example = "0d64c132-3aea-11ec-8d3d-0242ac130003")
+	private String contactSettingId;
+
+	@Schema(description = "Filter used by this delagate")
+	private Filter filter;
```

Comment:
Should this also be: `Map<String, List<String>>` ?

Reply:
Isn't List<Filter> sufficient?